### PR TITLE
JavaScript capstone project 

### PR DESCRIPTION
**Pull request by Maxwell011**
### Hello In this PR we did the following 👍 

- APIs
    - First, I used API so I could base the development of the web app around it. The API  allows me to:
            - Get a list of items with a unique item id (or generate the unique id).
            - The APIs I used were:
                  - TVmaze API: data about TV series and movies.
    - I used  Involvement API to record the different user interactions (likes, comments).
    
### Interfaces

- We build these interfaces:
     - The home page.
     - The comments popup.
     
### Home page

- When the page loads, the web app retrieves data from:
      - The selected API shows the list of items on the screen.
      -  The Involvement API to show the item likes.
- Remember that my page makes only 2 requests:
      - One to the base API.
      - And one to the Involvement API.
- When the user clicks on the Like button of an item, the interaction is recorded in the Involvement API and the screen is updated.
- When the user clicks on the "Comments" button, the Comments popup appears.
- When the popup loads, the web app retrieves data from:
- The selected API shows details about the selected item.
- The Involvement API to show the item comments.
- When the user clicks on the "Comment" button, the data is recorded in the Involvement API and the screen is updated.
- When the popup loads, the web app retrieves data from:
- The selected API shows details about the selected item.
- Counters We have counters in all the interfaces that show:
    - The number of items (home).
    - The number of comments (comments popup).

Kindly Review 🙏

### CODE REVIEW

File: modules/popup.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        <li><b>Rating:</b>: ${mi.rating.average}</li>
+        <div class='show-comment'>
+          <div>
+          <h3 class = "class-heading"> All Comments (${this.countComments(
+      arr,
+    )})</h3>
+          <ul class ="D-comments">
+          </ul>
+        </div>
+      </ul>`;
```
Comment (elisha2kyakpo1) [inline modules/popup.js:60]: - [ ] Let's fix the comments count. When there are no comments, we should return 0. When a comment is added the counter should update the UI in real-time, currently, I had to refresh for me to see the count of comments.

js

File: README.md [hunk 2]
```diff
@@ -1,56 +1,79 @@
...
+
+## Live Demo
 
-## Live Demo (if available)
+[Live Demo Link](https://maxwell011.github.io/Chill-Movies/)
 
-[Live Demo Link](https://livedemo.com)
+## Video Presentation
 
+[Video link](https://drive.google.com/file/d/1rxG4KAP-8vSpmqXKvwIuSPHUzvQWvE4m/view?usp=sharing)
```
Comment (elisha2kyakpo1) [inline README.md:20]: - [ ] I was expecting to see a piece of code (not just words) you are proud of according to this project's instructions
`The video contains a highlight about a piece of the code the student finds interesting or is proud of making.`. Kindly record another video, this time remember to highlight a piece of code you are proud of.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @Maxwell011 and @samuelmoiwa, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

The app UI looks superb, keep it up!

## Highlights

- The PR title and description looks professional 👍
- Descriptive commit messages 👍
- You have a descriptive readme file 👍
- No linter errors ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Maxwell011** [comment]
### Hello kindly review I have fixed the changes requested

**Comment by elisha2kyakpo1** [comment]
@Maxwell011, I have seen your hard work, you have fixed everything. Submit for another review, I wish you the best.

**Review by sumairq — APPROVED** [review]
### Hi Team 



In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**